plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -496,6 +511,23 @@ else if (precision > Decimals.MAX_PRECISION || actualPrecision <= 0) {
         return Optional.empty();
     }
 
+    private static ColumnMapping oracleTimestampColumnMapping(TimestampType timestampType)
+    {
+        if (timestampType.getPrecision() <= TimestampType.MAX_SHORT_PRECISION) {
+            return ColumnMapping.longMapping(
+                    timestampType,
+                    oracleTimestampReadFunction(timestampType),
+                    oracleTimestampWriteFunction(timestampType),
+                    FULL_PUSHDOWN);
+        }
+        checkArgument(timestampType.getPrecision() <= MAX_ORACLE_TIMESTAMP_PRECISION, "Precision is out of range: %s", timestampType.getPrecision());
```

Comment:
I feel this check is redundant. Since we are verifying `timestampType.getPrecision()` in `oracleLongTimestampWriteFunction` and `oracleLongTimestampWriteFunction`.